### ISSUE TITLE
[enhancement] Add date to output text

### ISSUE THREAD

**Issue by swechsler** [mydumper/mydumper#631]
Loading a multi-terabyte database can take several days. It would be helpful to have the date added to the messages that mydumper/myloader print out (currently it only shows the time), since it's not always obvious when grepping the output after the fact.

**Comment by swechsler** [comment]
I personally need it because mydumper/myloader is part of a script that I use to automate the setup of replication. The script's (and thus mydumper's) output is already saved; I saw no need to add an additional log file.

**Comment by davidducos** [comment]
Hi @swechsler, I created a function to modify the output to show the date, but I really don't like it. I also tried to modify the format of the default function but I didn't find anything related to that. 
As we already have a log that writes the date and as I didn't find a way to change the default behavior, I'm going to proceed to close this ticket and discard your enhancement. This is something that I don't like but I have to, as I don't want to add or maintain pieces of code that I know that are not ok.
